RELEASE_NOTES.md

```diff
@@ -8,18 +8,50 @@
 [//]: # ( # Deprecated)
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
+⚠️ This is an alpha release. Don't use it to accept payments in your live environment.
+
+## Breaking changes
+- All classes in `com.adyen.checkout.action` are now in `com.adyen.checkout.action.core`. If you import the classes, you must update import statements.
+- For Components integrations, each payment component no longer handles 3D Secure 2 and WeChat Pay actions. To handle the actions, you must add dependencies for each action:
+    ```Groovy
+    implementation 'com.adyen.checkout:3ds2:YOUR_VERSION'
+    implementation 'com.adyen.checkout:wechatpay:YOUR_VERSION'
+    ```
+    Exceptions: `CardComponent` and `BcmcComponent` can handle the 3D Secure 2 action. They don't require the additional dependencies.
+
 ## New
-* Payment method: Cash App Pay.
```

Comment:
Shouldn't we mention Cash App since it's new for v5?

Reply:
I think we want v5 to always include whatever was released already in v4, so the v5 release notes are kind of a continuation of the v4 ones, not a separate thing.